### ISSUE TITLE
Gageplots Feedback - General

### ISSUE THREAD

**Issue by amedenblik** [USGS-WiM/StreamStats#1709]
More feedback may be added later! This is what we have received so far.

- [x] Add gage number to title of plots ([notes here](https://wim-usgs.slack.com/archives/C2PG1BC3S/p1690570119823459)) #1787
 - [ ] Disable hamburger button options except for View in Full Screen
      - See [these Highcharts settings](https://api.highcharts.com/highcharts/exporting.buttons.contextButton.menuItems)
 - [ ] Fix small issues in appearance when you click the "Print" button at bottom of modal
      - All flood stats lines are showing up -- we want it to just be the selected lines
 - [x] Add a collapsible section for each plot that lists the data sources #1787
      - [x] Say "Not available" if a data sources is not available for that site (example: 06177020 doesn't have daily streamflow)

**Comment by maggiejaenicke** [comment]
Add to all three plots: 

In chartConfig constructor: 
`exporting : { buttons: {contextButton: {menuItems: string[]}}}
`

In this.chartConfig in `createAnnualFlowPlot()` function:
```
               exporting: {
                    buttons: {
                        contextButton: {
                            menuItems: ['viewFullscreen']
                        }
                    }
                }
```

Other note: the export options do work well for the Annual Peak Streamflow plot so we could always evaluate which options to keep on a plot-by-plot basis.

**Comment by amedenblik** [comment]
Closed in favor of #1970 and #1971
